components/org.wso2.scim2.testsuite.endpoint/src/main/webapp/src/components/Assertion.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import { Divider } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '90%',
+    margin: 10,
+    borderRadius: 25,
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: 550,
+  },
+  secondaryHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    color: theme.palette.text.secondary,
+  },
+  column: {
+    flexBasis: '100%',
+  },
+}));
+
+export default function Assertion(props) {
+  const classes = useStyles();
+  const [assertionData, setAssertionData] = React.useState();
+
+  return (
+    <div>
+      <Accordion elevation={0}>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel2a-content"
+          id="panel2a-header"
+          style={{ flexDirection: 'row-reverse', marginLeft: 25 }}
```

Comment:
It's better to avoid inline styling and use a separate style file. I think Material UI uses JSS and you should be able to do it using JS/TS alone.